src/client/containers/404Page/404Page.component.js

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { AppHeader } from '../../components/AppHeader/AppHeader.component';
+import Vector from '../../components/AppHeader/Vector.svg';
```

Comment:
The path appears to be incorrect here  the folder name inside components is 'Appheader'

Reply:
thanks @Kiran-Serolla , the storybook build have passed because of that. i tried a lot of things but it keeps failing.